### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vidurvij-Unity**
Problem:
https://github.com/Unity-Technologies/URDF-Importer/issues/13
The problem stems from the issue linked above. The instructions on the URDF Importer page were deprecated.
Solution:
Added basic instructions to use the package o the repo's front page. Plus added link to the main tutoial.

### CODE REVIEW

**Review by mpinol — COMMENTED** [review]
https://github.com/Unity-Technologies/Unity-Robotics-Hub/blob/main/tutorials/urdf_importer/urdf_tutorial.md#setting-up-the-urdf-importer-in-unity-editor has a link leading back to this read me that says it contains information on how to integrate the URDF importer. 

Should we include the instructions for adding this repo using package manager here?

**Review by mrpropellers — APPROVED** [review]
I'm fine with these changes pending resolution of Michael's comment. Be sure to add a small description of the change in the merge commit that includes: 1) What the problem was, 2) what you did to fix it, and 3) how you tested to confirm it's fixed (not really applicable in this case)

File: README.md [hunk 1]
```diff
@@ -2,17 +2,28 @@
...
+# Using the Package
 
-- Clone or download the URDF Importer Repo
-- In your Unity project create a directory named `Plugins` in `Assets`
-- Copy the `URDFLibrary` and `UnityEditorScripts` directories into the newly created `Assets/Plugins` directory
-- Confirm the integration succeded by selecting `Assets` from the menu bar and look for the `Import Robot from URDF` option
+## Adding the URDF package
 
-# Importing a URDF into a Unity scene
+1. Create a new Project in your Unity Hin and open the Unity Editor.
```
Comment (mpinol) [inline README.md:9]: `Unity Hin` to `Unity Hub`

File: README.md [hunk 2]
```diff
@@ -2,17 +2,38 @@
...
-# Importing a URDF into a Unity scene
+1. Create a new Project in your Unity Hub and open the Unity Editor.
+
+2. Open the Package Manager from Unity Menu. Click `Window -> Package Manager`. A new package manager window will appear.
+
+3. Click on the `+` sign on the top left corner of the package manager window and click on `Add Package from Git URL`. 
+
+![](https://github.com/Unity-Technologies/Unity-Robotics-Hub/blob/main/images/Package_manager_add.png)
+
+4. Paste the link to the URDF github, https://github.com/Unity-Technologies/URDF-Importer repository in the text box and press `Enter`.
```
Comment (mpinol) [inline README.md:17]: `.git` is required when pasting the link into package manager.

`https://github.com/Unity-Technologies/URDF-Importer.git`

File: README.md [hunk 3]
```diff
@@ -2,17 +2,38 @@
...
+
+2. Open the Package Manager from Unity Menu. Click `Window -> Package Manager`. A new package manager window will appear.
+
+3. Click on the `+` sign on the top left corner of the package manager window and click on `Add Package from Git URL`. 
+
+![](https://github.com/Unity-Technologies/Unity-Robotics-Hub/blob/main/images/Package_manager_add.png)
+
+4. Paste the link to the URDF github, https://github.com/Unity-Technologies/URDF-Importer repository in the text box and press `Enter`. 
+
+4. Paste the link to the URDF github repository,  in the text box and press `Enter`.
```
Comment (mpinol) [inline README.md:19]: Same instruction as above.

File: README.md [hunk 4]
```diff
@@ -2,17 +2,38 @@
...
+# Using the Package
 
-- Clone or download the URDF Importer Repo
-- In your Unity project create a directory named `Plugins` in `Assets`
-- Copy the `URDFLibrary` and `UnityEditorScripts` directories into the newly created `Assets/Plugins` directory
-- Confirm the integration succeded by selecting `Assets` from the menu bar and look for the `Import Robot from URDF` option
+## Adding the URDF package
 
-# Importing a URDF into a Unity scene
+1. Create a new Project in your Unity Hub and open the Unity Editor.
```
Comment (mpinol) [inline README.md:9]: Do we need this point or can we prepend something like, 'In the Unity Editor,' to the second point?

**Review by mpinol — DISMISSED** [review]

**Review by mpinol — APPROVED** [review]

**Pull request merged**